filebeat/_meta/fields.common.yml

```diff
@@ -120,6 +120,36 @@
       description: >
         IGMP type.
 
+
+    - name: azure-eventhub
```

Comment:
This name is used as part of the mapping, resulting events put all these fields under `azure` key

```suggestion
    - name: azure
```

Reply:
indeed, should be under azure, fixed the issue and regenerated the files